src/core/client/stream/common/UserBox/UserBoxContainer.tsx

```diff
@@ -43,11 +44,20 @@ export class UserBoxContainer extends Component<Props> {
   private handleRegister = () => this.props.showAuthPopup({ view: "SIGN_UP" });
   private handleSignOut = () => this.props.signOut();
 
+  private get authTypePreventsLogout() {
+    const integrations = getAuthenticationIntegrations(
+      this.props.settings.auth,
+      "stream"
+    );
+    return integrations.includes("sso") && !integrations.includes("local");
```

Comment:
This should instead be:

```suggestion
    return integrations.length === 1  && integrations[0] === "sso";
```